CMakeLists.txt

```diff
@@ -74,6 +74,8 @@ include(InstallRequiredSystemLibraries)
 # Allow the user to decide whether to build the shared libaries or the static libraries.
 option(BUILD_SHARED_LIBS "Build shared libraries" ON)
 set(BUILD_VERSION "0.0.0" CACHE STRING "Library version (for both libbsoncxx and libmongocxx)")
+# Allow the user to use or ignore a bundled C driver (i.e., one included via add_subdirectory)
```

Comment:
If the user already did an `add_subdirectory` of the mongo c driver, is there any circumstance where you think they *wouldn't* want to use it?

Reply:
Such a situation might occur in a large multi-project build, though I have to imagine that it would be rare.  This option doesn't seem like it would cause any problems, but if it is unlikely to be used, then it might be better eliminated.  @kevinAlbs do you have any thoughts one way or the other on this?

Reply:
In our discussions, I don't think we had a strong reason either way. Though I do suppose since users can't bundle C++ driver currently, we could skip adding this option now, and see if any users do ask for it later for that rare use case. That seems slightly more preferable. Though I'm not opposed to having the option upfront either.